docs/content/examples/alt-text.html

```diff
@@ -0,0 +1,152 @@
+---
+layout: page
+title: Alt-text
+description: Everything we know about alt-texts, when to use them, and how to craft them.
```

Comment:
Plural feels a little weird to me, though I'm sure it's correct, I've basically never seen plural used for alt-text. Feel free to ignore this nit.